contracts/transfer/Makefile

```diff
@@ -1,12 +1,6 @@
-.PHONY: all help host hosted 
-
-all: host
-
-check: hosted ## Run the Rust check on the project features
-	@cargo check --target wasm32-unknown-unknown
-	@cargo check
+.PHONY: wasm check test help
```

Comment:
As said, unless there is a strong reason to have this at the top, revert it to be at the bottom.

Also, add the `help` common recipe.